OutOfSchool/OutOfSchool.BusinessLogic/Config/Elasticsearch/ElasticsearchCompetitiveEventConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Elastic.Clients.Elasticsearch.Analysis;
+using Elastic.Clients.Elasticsearch.IndexManagement;
+using OutOfSchool.ElasticsearchData.Extensions;
+
+namespace OutOfSchool.BusinessLogic.Config.Elasticsearch;
+
+/// <summary>
+/// Contains methods to configure <see cref="CompetitiveEventES"/> model into Elasticsearch index.
+/// </summary>
+public class ElasticsearchCompetitiveEventConfiguration : IElasticsearchEntityTypeConfiguration<CompetitiveEventES>
```

Comment:
this comment is not a direct call to action, but we can do a followup task to extract ES configuration classes, Index creation and pinging hosted services to the ES subproject as they do not interact directly with any business logic objects.